src/views/open-api-docs-view/utils/get-response-data.ts

```diff
@@ -0,0 +1,72 @@
+// Utils
+import { getPropertyDetailPropsFromSchemaObject } from './get-property-detail-props'
+// Types
+import type { PropertyDetailsProps } from '../components/property-details'
+import type { OpenAPIV3 } from 'openapi-types'
+
+/**
+ * Map known response codes to human-readable headings.
+ *
+ * If an response code isn't in this map, we use the response code itself
+ * as the heading.
+ */
+const RESPONSE_CODE_HEADINGS: Record<string, string> = {
+	'200': 'Successful Response',
+	default: 'Default Response',
+}
+
+/**
+ * Given OpenAPI responses data,
+ * Return response data formatted for display.
+ */
+export async function getResponseData(
```

Comment:
As with `get-request-data`, a lot of the work in `getResponseData` is being thorough in respecting types, and parsing out the parts of the schema we want.